### PR TITLE
Blink assignment with comments

### PR DESCRIPTION
**Pull request by dvasisht**
I've included comments for the jquery for Blink as per your request

### CODE REVIEW

**Comment by dvasisht** [comment]
I've included the comments for the Blink assignment

**Comment by dvasisht** [comment]
"The Travis CI build failed. This pull request can be automatically merged, but the status isn't passing." Aidan, are you able to see the code with the comments?

**Comment by afeld** [comment]
Yep, click through the Travis link to see the errors (just a couple minor things):

https://travis-ci.org/advanced-js/blink/builds/29174895

File: blink.js [hunk 1]
```diff
@@ -1 +1,39 @@
-// YOUR CODE GOES HERE
+//Specifies the function to run after the document is loaded
```
Comment (afeld) [inline blink.js:1]: That's what the `$(document).ready()` is doing below... what is line 2+32 doing?

File: blink.js [hunk 2]
```diff
@@ -1 +1,39 @@
...
+    {
+        var defaults =
+        {
+            delay: 500
+        };
+
+		// Merges the contents of two or more objects together into the first object
+        var options = $.extend(defaults, options);
+
+		//On each return, calls an anonymous function that would set the interval to toggle between hiding the object and making it visible
```
Comment (afeld) [inline blink.js:15]: What is it iterating through?

**Comment by afeld** [comment]
Asked some questions above, but the code looks good :+1:

**Comment by afeld** [comment]
(feel free to simply respond in a comment through the web interface)

**Comment by afeld** [comment]
Moved over to https://github.com/advanced-js/blink/pull/19.

**Pull request closed without merging**